core/lib/spree/testing_support/factories/adjustment_factory.rb

```diff
@@ -29,7 +29,7 @@
       after(:create) do |adjustment|
         # Set correct tax category, so that adjustment amount is not 0
         if adjustment.adjustable.is_a?(Spree::LineItem)
-          adjustment.source.tax_category = adjustment.adjustable.tax_category
+          adjustment.source.tax_categories = [adjustment.adjustable.tax_category]
```

Comment:
I wonder if this should be `adjustment.source.tax_categories << adjustment.adjustable.tax_category`?  We could leave that as a question to ask people in the solidusio/solidus PR perhaps.